WcaOnRails/app/views/competitions/my_competitions.html.erb

```diff
@@ -49,7 +49,22 @@
   </h3>
 
   <%= t '.bookmarked_explanation' %>
-
+  <%= form_tag('/competitions/mine', method: :get, class: "competition-select form-inline #{@display}", id: "bookmarked-competitions-query-form") do %>
+    <div id="registration-status" class="form-group registration-status-selector">
+      <%= check_box_tag(:show_registration_status, params[:show_registration_status], params[:show_registration_status] == "on") %>
+      <%= label_tag(name = :show_registration_status, content_or_options = t('competitions.index.show_registration_status')) %>
+    </div>
+  <% end %>
   <%= render "my_competitions_table", competitions: @bookmarked_competitions, registrations_by_competition_id: @registered_for_by_competition_id, past: false, bookmarked: true %>
 
 </div>
+
+<script>
+$(document).ready(function() {
+    function submitForm() {
+      var $form = $('#bookmarked-competitions-query-form');
```

Comment:
Please bind the jQuery object outside of the `submitForm` function. You can take a look at `app/assets/javascripts/competitions.js` ll.94 for inspiration.